scripts/npc/2042002_2.js

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
i think this was the old version of the npc script before i overwrote it, i wanted to keep it in case we needed it for something but so much has changed in the master branch i dont think it is necessary anymore